src/lights/LightComposite.cpp

```
@@ -1,13 +1,16 @@
+    const Math::Vector3D& normal, const Math::Vector3D& objectColor,
+    const Math::Point3D& hitPoint, const ShapeComposite& shapes) const {
   Math::Vector3D result(0, 0, 0);
   for (const auto& light : _lights) {
-    result = result + light->computeLighting(normal, object_color);
+    result += light->computeLighting(normal, objectColor, hitPoint, shapes);
   }
   return result;
 }
 
-void Raytracer::LightComposite::addLight(const std::shared_ptr<ILight> &newLight) {
+void Raytracer::LightComposite::addLight(
```

Cppcheck (2.10-2)

[style] The function 'addLight' is never used.

(unusedFunction)